lib/upload.js

```diff
@@ -1,262 +1,280 @@
-let fingerprint = require("./fingerprint")
-let extend = require("extend")
-
-let defaultOptions = {
-    endpoint: "",
-    fingerprint,
-    resume: true,
-    onProgress: null,
-    onSuccess: null,
-    onError: null,
-    headers: {},
-    chunkSize: Infinity,
-    withCredentials: false
-}
+/* global window */
+import fingerprint from './fingerprint';
```

Comment:
nope, no reason. we should stick to only one way. made it single quotes in https://github.com/fahad19/tus-js-client/commit/e5f80c0889a386751624040f62607372f01da22d